pkg/cli/schemaherocli/apply.go

```diff
@@ -26,18 +26,17 @@ func Apply() *cobra.Command {
 			driver := v.GetString("driver")
 			ddl := v.GetString("ddl")
 			uri := v.GetString("uri")
-			vaultUriRef := v.GetString("vault-uri-ref")
```

Comment:
vault-uri-ref was abstracted one level higher, and all plans and apply are passed the generated uri